### PR TITLE
feat: add width, height as props to App.tsx

### PR DESCRIPTION
**Pull request by ad1992**
Width and height are received as props.
cc @dwelle

### CODE REVIEW

**Comment by ad1992** [comment]
@dwelle did you get a chance to check this?

**Comment by dwelle** [comment]
Lately I don't have any time, and there's a lot in my backlog I need to review on Excalidraw. Might take a bit. That said, this looks small enough that I may wanna do it first.

File: src/components/App.tsx [hunk 1]
```diff
@@ -194,13 +195,22 @@ class App extends React.Component<any, AppState> {
...
   constructor(props: any) {
     super(props);
+    const defaultAppState = getDefaultAppState();
+
+    const { width, height } = props;
+    this.state = {
+      ...defaultAppState,
+      width,
+      height,
+    };
```
Comment (dwelle) [inline src/components/App.tsx:212]: why replacing `state` with indirect `defaultProps` which then must be spread in the constructor? Also, you got rid of the `isLoading: true` in the process.

I'd keep `public state`, and just add the width/height there. Were you solving some problem?

Comment (ad1992) [reply src/components/App.tsx:212]: I didn't get you, I am checking old width prop and new width prop when updating the width state, when do you think the state .width will not reflect the props.width coz componentDIdUpdate will always get triggered when prop updated ? could you explain more?

Comment (dwelle) [reply src/components/App.tsx:212]: nvm, I forgot how classical React works :).

Add the `isLoading: true` in the constructor and we can merge.

**Pull request merged**